blog/2023-02-15-netdata-functions.md

```diff
@@ -0,0 +1,72 @@
+
+Think of it as a toolkit that allows you to quickly and easily retrieve specific information about your infrastructure, without having to SSH into each node individually and run commands to retrieve that information. With functions, you can gather more information and perform actions on a given node without having to access it directly. This can save SysAdmins and DevOps engineers time and effort, especially when managing large clusters of nodes.
+
+If you'd rather see it in action than read about it, visit our [demo space](https://app.netdata.cloud/spaces/netdata-demo/rooms/all-nodes/functions) and see it in action, live!
+
+### What can functions do? 
+
+In theory, functions can do basically anything that you want. But while we are thrilled to introduce the Functions feature, we are doing so cautiously, starting with just one supported function. We understand that invoking a routine or script on a node carries security implications, which is why we have taken great care to ensure that the initial function is secure and does not compromise your infrastructure. As we continue to develop new functions, we will take the same approach, prioritizing security to ensure the safety of our users' infrastructure.
+
+So to start with, there's one function: "Processes"
+
+### The "Processes" function
+
+This function provides a tabular view of all the processes running on a system, along with an extensive range of information for each process. These details include the process ID (PID), command name (Cmd), category, user, and various usage statistics such as CPU, memory usage, I/O, network, and more.
+
+By having access to a comprehensive overview of all the processes running on a system, it is much easier to identify any issues that may arise, such as a process consuming too much CPU or memory. 
+
+The "Processes" function allows users to monitor multiple nodes with just a single click. This feature saves a considerable amount of time and effort as it eliminates the need for users to SSH into each node and run commands to retrieve this data. Instead, all the data can be accessed and viewed directly from the Netdata dashboard.
+
+![image](https://user-images.githubusercontent.com/24860547/219148788-393d42d3-ef34-43bd-bb8b-e612246923c5.png)
+
+Let's take a look at a real-world example of how using the "Processes" function can help solve a particular problem.
+
+Imagine you have a server that's been acting up recently. You're not sure why, but you suspect it might be due to a rogue process that's consuming too many resources. Previously you had to observe multiple different charts to get the complete picture of what was going on, but with the "Processes" function, you can easily get a detailed breakdown of all the processes running on the node, including CPU and memory usage. From there, you can identify the culprit and take action to resolve the issue.
```

Comment:
```suggestion
Imagine you have a server that's been acting up recently. You're not sure why, but you suspect it might be due to a rogue process that's consuming too many resources. Previously you had to observe multiple different charts to get the complete picture of what was going on, and often multiple processes could be combined into a category making it harder to quickly identify the culprit. 

But with the "Processes" function, you can easily get a detailed breakdown of all the processes running on the node, including CPU and memory usage. From there, you can identify the culprit and take action to resolve the issue.
```